modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
```

Comment:
This is true. I wasn't sure whether to put under `/var` or `/opt`, but I saw we already use `/opt` for a few things, like stats, mirrors, and apt, so I just went with that. i'll take your suggestion, @chriskuehl.